### PR TITLE
Reconnect failed tab.

### PR DESCRIPTION
**Pull request by maxhora**
Fixes https://github.com/jumptrading/FluentTerminal/issues/222

Overlay to reconnect/close tab; Shortcut to reconnect tab.

### CODE REVIEW

**Comment by felixse** [comment]
I'm not a fan of the UI part of this. The tab item is not very easy to discover, plus this adds an almost always disabled item to the menu (which is also quite redundant for non ssh tabs).

How about some overlay similar to search/resize with two buttons (reconnect, close)?

**Comment by MaxRis** [comment]
@felixse sounds reasonable, will try to update this PR with new overlay changes

File: FluentTerminal.App/Views/TerminalView.xaml [hunk 1]
```diff
@@ -18,111 +18,149 @@
...
+                        ToolTipService.ToolTip="Close search panel">
+                        <Button.Content>
+                            <Viewbox Width="16" Height="16">
+                                <SymbolIcon Symbol="Cancel" />
+                            </Viewbox>
+                        </Button.Content>
+                    </Button>
+                </StackPanel>
+            </controls:DropShadowPanel>
+            <controls:DropShadowPanel
```
Comment (maxhora) [inline FluentTerminal.App/Views/TerminalView.xaml]: @felixse  New `DropShadowPanel` is wrapped into StackPanel along with overlay for Search.

**Comment by maxhora** [comment]
@felixse replaced new context menus with two buttons overlay to reconnect or close failed tab. Please let me know if any further adjustments make sense, thanks!

**Pull request merged**